storage/innobase/include/os0file.h

```diff
@@ -338,7 +338,7 @@ struct Encryption {
     KEYRING = 2
   };
 
-  enum Encryption_rotation { NO_ROTATION, MASTER_KEY_TO_KEYRING };
+  enum class Encryption_rotation : char { NO_ROTATION, MASTER_KEY_TO_KEYRING };
```

Comment:
unsigned char or std::uint8_t would be better

You could also move it outside of Encryption, that would make the value references shorter.

Reply:
done